asabiris/output/smtp/service.py

```
@@ -53,6 +65,16 @@ def __init__(self, app, service_name="SmtpService", config_section_name='smtp'):
 		self.Subject = asab.Config.get(config_section_name, "subject")
 		self.ValidateCerts = asab.Config.getboolean(config_section_name, "validate_certs", fallback=True)
 		self.Cert = asab.Config.get(config_section_name, "cert_bundle", fallback="").strip()
+		self.ProxyHost = asab.Config.get(config_section_name, "proxy_host", fallback="").strip()
+		self.ProxyPort = asab.Config.get(config_section_name, "proxy_port", fallback="").strip()
+		self.ProxyUser = asab.Config.get(config_section_name, "proxy_user", fallback="").strip()
+		self.ProxyPassword = asab.Config.get(config_section_name, "proxy_password", fallback="")
+		self.ProxyConnectTimeout = asab.Config.getint(config_section_name, "proxy_connect_timeout", fallback=10)
+
+		if self.ProxyHost and not self.ProxyPort:
+			raise ValueError("SMTP proxy is configured but `proxy_port` is empty")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)